domains/roxza.json

```diff
@@ -7,6 +7,6 @@
     "discord": "Roxza#0002"
   },
   "record": {
-    "CNAME": "roxza.me"
+    "CNAME": "https://roxza.me"
```

Comment:
The `CNAME` cannot have any slashes, if you want the domain to just redirect to your website, you can use a `URL` record instead.

```
```